src/components/CardCar/CardCar.tsx

```diff
@@ -0,0 +1,46 @@
+import { ICar } from "../../interfaces/carInterface";
+import { ContainerCardCar } from "./style";
+
+const CardCar = () => {
+  const car: ICar = {
+    brand: "Mercedes Benz",
+    model: "A 200 CGI ADVANCE SEDAN",
+    year: 2013,
+    km: 0,
+    color: "Branco",
+    isPromo: true,
+    price: "22.000",
+    description: "é um carro",
+    isActive: true,
+    createdAt: "2023-06-04T16:17:49.041Z",
+    updatedAt: "2023-06-04T16:17:49.041Z",
+    id: "hiheiuhieuhaiheuaheiheuihiauhuieh"
+  };
+
+  return (
+    <ContainerCardCar>
+      <div className="container-car-name">
+        <h1>{`${car.brand} ${car.model} `}</h1>
+      </div>
+      <div className="container-car-info">
+        <ul className="container-car-info">
+          <li key={car.id}>
+            <div className="car-tag">
+            <span>{car.year}</span>
+            <span>{car.km} KM</span>
+            </div>
+            
+            <h3>R$ {car.price}</h3>
```

Comment:
ficou perfeito o componente, mas só falta colocar um toFixed e substituir o ponto decimal por vírgula, ex: R$ 1532,00

Reply:
The adjustment was made to the car price tag